.github/workflows/publish-results.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish results
+
+on:
+  workflow_run:
+    workflows: ["Benchmark"]
+    types:
+      - completed
+
+jobs:
+  publish_results:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - name: Install Requirements
+        run: pip install -r requirements.txt
+      - name: Copy asv.conf
+        run: cp -f asv.conf.json ci-results/
+      - name: Publish results
+        run: |
+          cd ci-results
+          asv publish
```

Comment:
Can we use `asv gh-pages` to publish the pages? I can't remember off hand if you need to run `asv publish` first or if the `gh-pages` command builds and pushes it.

https://asv.readthedocs.io/en/stable/commands.html#id18